src/Tickets/Emails/Admin/Emails_Tab.php

```diff
@@ -172,4 +201,122 @@ public function get_fields(): array {
 		 */
 		return apply_filters( 'tec_tickets_emails_settings_fields', $fields );
 	}
+
+	/**
+	 * Check if currently editing email.
+	 * 
+	 * @since TBD
+	 *
+	 * @param Email_Abstract $email
+	 * 
+	 * @return boolean
+	 */
+	public function is_editing_email( $email = null ) {
+		if ( ! $this->is_on_section() ) {
+			return false;
+		}
+
+		// Get `section` query string from URL.
+		$editing_email  = tribe_get_request_var( 'section' );
+
+		// If email wasn't passed, just return whether or not string is empty.
+		if ( empty( $email ) ) {
+			return ! empty( $editing_email );
+		}
```

Comment:
This is what I mention that we can rely on here https://github.com/the-events-calendar/event-tickets/pull/2555/files#r1119849253